schema/PACK.xsd

```diff
@@ -1432,8 +1434,11 @@
       <xs:element name="mountedPart"      type="MountedPartType"      minOccurs="0" maxOccurs="unbounded" />
       <xs:element name="image" minOccurs="0">
         <xs:complexType>
+          <!-- attribute 'large' displays the 'top' side of the board and is required -->
+          <xs:attribute name="large" type="xs:string" use="required" />
```

Comment:
@silviooliva it looks like we do have published board descriptions that do only specify "large" and some only "small", therefore we are forced to make all attributes "optional".

Reply:
Thanks a lot for switching back to "optional".